events/api/src/main/java/org/projectnessie/events/api/Event.java

```diff
@@ -51,6 +52,13 @@ public interface Event {
    */
   UUID getId();
 
+  /** The id of the event, as a string for convenience. */
+  @Value.Lazy
```

Comment:
Why do we need this in general and as a memoized field?

Reply:
Creating the string representation of an UUID is a bit costly. But in many places inside Nessie, we need that string. Memoizing it saves us a few CPU cycles. I think this method could be useful for subscribers too (think: adding headers to a Kafka record for instance).